Cloud/NXDOC-2261: Review nuxeo.conf page (#1697)
nuxeo/doc.nuxeo.com-content: src/nxdoc/nuxeo-server/administration/variables-and-configuration/configuration-parameters-index-nuxeoconf.md

_Default value:_ 60000

#### `nuxeo.server.ajp.enabled`

> Enable Tomcat AJP connector

Reviewer: ```suggestion
Enable Tomcat AJP connector.
```

_Default value_: false

#### `nuxeo.server.ajp.port`

Server AJP listen port.

_Default value:_ 8009

#### `nuxeo.server.cookies.sameSite`

**Since Nuxeo 10.10-HF34, Nuxeo 11.4 and Nuxeo LTS 2021**

Allows setting the same site cookie policy.

> Possible values: none, lax or strict

Reviewer: ```suggestion
Possible values: none, lax or strict.
```

*
{{! /multiexcerpt}}

#### `templateName.include` - only available in template properties

> Templates to inlcude prior to the one being included.

Reviewer: ```suggestion
Templates to include prior to the ones being included.
```

Author: Keeping one as it is relative to the `templateName`.
